README.md

```diff
@@ -556,3 +558,27 @@ yarn start
 ## Stats
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/regular-table) 
 -->
+
```

Comment:
This is already in the [CONTRIBUTING.md](https://github.com/finos/regular-table/blob/master/CONTRIBUTING.md) which has been the convention for programming project for decades.

Reply:
removed